docs/METRICS.md

```diff
@@ -311,8 +311,9 @@ These are events that an add-on user can encounter on a shot they own
 33. [x] Visit an image directly, when the image isn't embedded directly in a Screenshots shot page, `web/visit/direct-view-owner`
 34. [x] View an image directly, when the image is being shown as part of a Facebook/Twitter style preview (the og:image or twitter:image), `web/visit/direct-view-embedded-owner`
 35. [x] Close new edit tools promotion dialog, `web/promo-closed`
-1. [x] Add shot to favorites `web/set-favorite/navbar`
-1. [x] Remove shot from favorites `web-unset-favorite/navbar`
+36. [x] Add shot to favorites `web/set-favorite/navbar`
+37. [x] Remove shot from favorites `web-unset-favorite/navbar`
+38. [x] Signin to Firefox Accounts `web/fxa-signin`
```

Comment:
Markdown doesn't care about the actual number in the numbered lists btw.

Reply:
true, I believe its more for readability to increase number for a items listed in a group